src/Orckestra.Composer.Cart/Services/Order/OrderHistoryViewService.cs

```diff
@@ -95,6 +94,30 @@ public virtual async Task<OrderHistoryViewModel> GetOrderHistoryViewModelAsync(G
             {
                 shipmentsTrackingInfos = await GetShipmentsTrackingInfoViewModels(orderQueryResult, param).ConfigureAwait(false);
             }
+            var orderSettings = await GetOrderSettings(param.Scope).ConfigureAwait(false);
+            
+            var fulfillmentOrdersList = new List<FulfillmentOrderQueryResult>();
+            if (param.OrderTense == OrderTense.CurrentOrders)
```

Comment:
also need to validate next
orderQueryResult != null && orderQueryResult.Results != null && param.OrderTense == OrderTense.CurrentOrders

Probaly you can move this code to line 96 - it already has this check